<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1281</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/fondos/Ingresar.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="imagenes.qrc">:/fondos/Ingresar.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>590</y>
        <width>541</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="imagenes.qrc">:/fondos/main.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxOrigen">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 11pt &quot;Nirmala UI&quot;;
border: rgb(255, 255, 255);
color: rgb(0, 64, 143);</string>
      </property>
     </widget>
     <widget class="QLabel" name="origen">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 16pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Origen:</string>
      </property>
     </widget>
     <widget class="QLabel" name="destino">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 16pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Destino:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxDestino">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 11pt &quot;Nirmala UI&quot;;
border: rgb(255, 255, 255);
color: rgb(0, 64, 143);</string>
      </property>
     </widget>
     <widget class="QLabel" name="tipoDe">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 16pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Tipo de</string>
      </property>
     </widget>
     <widget class="QLabel" name="transporte">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 16pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Transporte:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTipoTransporte">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 11pt &quot;Nirmala UI&quot;;
border: rgb(255, 255, 255);
color: rgb(0, 64, 143);</string>
      </property>
      <item>
       <property name="text">
        <string>Vehículo Particular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Transporte Público</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="restablecer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 14pt &quot;Segoe UI Black&quot;;
background-color: rgb(237, 255, 241);</string>
      </property>
      <property name="text">
       <string>RESTABLECER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>630</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CALCULAR RUTA</string>
      </property>
     </widget>
     <widget class="QLabel" name="origen_2">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 16pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>DETALLES: </string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>-&gt; Distancia:</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_2">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>160</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>-&gt; Tiempo</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_3">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>190</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Estimado:</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_4">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>-&gt; Consumo de</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_5">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>300</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Combustible aprox:</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_6">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>380</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>-&gt; Paradas</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_7">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>410</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Intermedias:</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_8">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>490</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>-&gt; Rutas</string>
      </property>
     </widget>
     <widget class="QLabel" name="distancia_titulo_9">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>520</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Alternas:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textAlternas">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>560</y>
        <width>171</width>
        <height>141</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="distancia">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>110</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>0 KM</string>
      </property>
     </widget>
     <widget class="QLabel" name="tiempo">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>0 horas</string>
      </property>
     </widget>
     <widget class="QLabel" name="combustible">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>0 L.</string>
      </property>
     </widget>
     <widget class="QLabel" name="paradas">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>440</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>0 paradas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recorrido">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>630</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Segoe UI Black&quot;;
background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>VER EL RECORRIDO</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxVelocidad">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>660</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 11pt &quot;Nirmala UI&quot;;
border: rgb(255, 255, 255);
color: rgb(0, 64, 143);</string>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="origen_3">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>620</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
font: 87 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Velocidad:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCARGAR">
    <property name="title">
     <string>Cargar Mapa</string>
    </property>
    <addaction name="actionCARGAR"/>
   </widget>
   <widget class="QMenu" name="menuCONFIGURACION">
    <property name="title">
     <string>Configuracion</string>
    </property>
    <addaction name="actionCAMBIAR_CONDICIONES"/>
    <addaction name="actionCAMBIAR_COLOR_A_LAS_RUTAS"/>
    <addaction name="actionPRIORIZAR_RUTAS_CORTAS"/>
   </widget>
   <widget class="QMenu" name="menuAYUDA">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionGUIA"/>
   </widget>
   <addaction name="menuCARGAR"/>
   <addaction name="menuCONFIGURACION"/>
   <addaction name="menuAYUDA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCARGAR">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionCAMBIAR_CONDICIONES">
   <property name="text">
    <string>Cambiar Condiciones</string>
   </property>
  </action>
  <action name="actionCAMBIAR_COLOR_A_LAS_RUTAS">
   <property name="text">
    <string>Cambiar color de las rutas</string>
   </property>
  </action>
  <action name="actionPRIORIZAR_RUTAS_CORTAS">
   <property name="text">
    <string>Priorizar rutas cortas</string>
   </property>
  </action>
  <action name="actionGUIA">
   <property name="text">
    <string>Guia</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
